### PR TITLE
Hb 7

### PR DESCRIPTION
**Pull request by ilya449**

### CODE REVIEW

File: src/main/java/com/cinema/model/Order.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @NonNull
+    @OneToMany
+    private List<Ticket> tickets;
+    @NonNull
+    private LocalDateTime orderDate;
+    @NonNull
+    @ManyToOne
```
Comment (dariiabutorina) [inline src/main/java/com/cinema/model/Order.java]: Screenshot 2020-10-12 at 13 08 48

File: src/main/java/com/cinema/dao/impl/OrderDaoImpl.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    public List<Order> getOrderHistory(User user) {
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            return session.createQuery("from Order o "
+                    + "join fetch o.tickets "
+                    + "where o.user = :user", Order.class)
+                    .setParameter("user", user)
+                    .getResultList();
+        } catch (Exception e) {
+            throw new DataProcessingException("Couldn't get shopping cart of user: " + user, e);
+        }
```
Comment (dariiabutorina) [inline src/main/java/com/cinema/dao/impl/OrderDaoImpl.java]: Is this catch block necessary?

**Review by dariiabutorina — CHANGES_REQUESTED** [review]
Good job :)

File: src/main/java/com/cinema/model/Order.java (hunk 1 above)
Comment (ilya449) [reply src/main/java/com/cinema/model/Order.java]: fixed

File: src/main/java/com/cinema/dao/impl/OrderDaoImpl.java (hunk 2 above)
Comment (ilya449) [reply src/main/java/com/cinema/dao/impl/OrderDaoImpl.java]: is not, ty

File: src/main/java/com/cinema/model/Order.java (hunk 1 above)
Comment (dariiabutorina) [reply src/main/java/com/cinema/model/Order.java]: Sorry for confusing
This is redundant
It was about tickets, movies etc, not users 
My apologies

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**